src/apis/index.js

```diff
@@ -0,0 +1,11 @@
+import { BACKEND_BASE_URL } from "./constants";
+
+export const fetchPosts = async () => {
+  const url = `${BACKEND_BASE_URL}/posts`;
```

Comment:
you can also make it a generic function that receive the "url" parameter and return the respective response